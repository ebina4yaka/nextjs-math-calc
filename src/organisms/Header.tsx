import React from 'react'
import { AppBar, Toolbar, Typography } from '@material-ui/core'
import { makeStyles } from '@material-ui/core/styles'

type Props = {
  title: string
}

const useStyles = makeStyles({
  header: {
    width: '100%',
  },
})

export default function Header(props: Props): React.ReactElement {
  const classes = useStyles()
  const { title } = props
  return (
    <div className={classes.header}>
      <AppBar position="static">
        <Toolbar>
          <Typography variant="h6">{title}</Typography>
        </Toolbar>
      </AppBar>
    </div>
  )
}
