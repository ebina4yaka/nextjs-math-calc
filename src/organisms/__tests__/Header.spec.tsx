import React from 'react'
import renderer from 'react-test-renderer'
import Header from '../Header'

test('Header Snapshot Test', () => {
  const tree = renderer.create(<Header title="MathCalc" />).toJSON()
  expect(tree).toMatchSnapshot()
})
