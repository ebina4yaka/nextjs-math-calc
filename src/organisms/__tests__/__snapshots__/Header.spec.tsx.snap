// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header 1`] = `
<div
  className="makeStyles-header-1"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h6
        className="MuiTypography-root MuiTypography-h6"
      >
        MathCalc
      </h6>
    </div>
  </header>
</div>
`;
